x-pack/plugins/observability_solution/synthetics/common/rules/alert_actions.ts

```diff
@@ -128,6 +131,14 @@ export function populateAlertActions({
           actions.push(recoveredAction);
         }
         break;
+      case OPSGENIE_ACTION_ID:
+        // @ts-expect-error
```

Comment:
Is it possible to avoid `@ts-expect-error`?

Reply:
i am clueless on how to handle it.